client/dist/index.html

```diff
@@ -2,12 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8"/>
-    <title>SERUM</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="styles.css">
+    <title>SERUM</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
```

Comment:
Don't think you want to load so many weights as it will slow down load. Probably only a regular weight and bold one, like 400 and 600 (or 700 or 800, depending on what the font it)

Reply:
I wanted to load everything we might need at first for prototyping/experimentation then slim it down once we've determined what we actually. I'll be sure to trim it though.